pkg/models/grt_feedback.go

```diff
@@ -0,0 +1,16 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// GRTFeedback models GRT Feedback
+type GRTFeedback struct {
```

Comment:
I feel like we're missing the type of feedback we're issuing. Unless you want to do that as part of the next story, which could also make sense.

When we get to the display portion, we'll need to know whether the feedback was for the business owner of for the governance review board. So, maybe we could add "audience" as a field? Not quite sure the best way to model that.